Doc/library/stackless/pickling.rst

```diff
@@ -108,6 +108,18 @@ different address than *t1*, which was displayed earlier.
     objects and frame objects contain code objects. And code objects are
     usually incompatible between different minor versions of |CPY|.
 
+.. note::
+
+    If you pickle a tasklet, its :class:`~contextvars.Context` won't be pickled,
+    because :class:`~contextvars.Context` objects can't be pickled. See
+    :pep:`567` for an explanation.
+
+    It is however possible to create a subclass of :class:`tasklet` and
+    overload the methods :meth:`tasklet.__reduce_ex__` and :meth:`tasklet.__setstate__` to
+    pickle the values of particular :class:`~contextvars.ContextVar` objects together
+    with a tasklet.
+
+
```

Comment:
This problem also affects pickling tasklets with a 
`(<class '_stackless.cframe'>, (), (1, 'context_run_callback', ((1, 2), <Context object at 0x7f59fe29ad60>, None, None), 1, 0))` on the stack. This cframe contains a Context.

Pickling Context objects has been discussed in PEP 567 and was rejected. I have to think about this problem.

Reply:
Resolved by 4148ec3c47

Reply:
I think I read in the context vars pep that pickling was not ignored, but might be addressed by a future Python version...
The reasons given were not very clear.  One very poor reason was that people might put unpicklable stuff into the context...
I honestly don't quite understand why they chose such a complicated way to do context vars, all with the immutability and stuff.